docs/src/understand/thin-edge-json.md

```diff
@@ -58,38 +113,38 @@ For example, a multi-level measurement as follows is NOT valid:
       "L3": 8.8
     }
   }
-}
+}'
 ```
 
 because the values at the second level(`phase1`, `phase2` and `phase3`) are not numeric values.
```

Comment:
This sentence might not be needed anymore if the bold text before the example already explains why the example is invalid (though in slightly different words)

Reply:
Removed